### PR TITLE
Comprehensive Deployment Guide for Setting Up an EKS Cluster

### PR DESCRIPTION
**Pull request by Rolika4**
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Pull Request contain one commit. I squash my commits.

### CODE REVIEW

File: docs/operator-guide/install-argocd.md [hunk 1]
```diff
@@ -87,106 +87,107 @@ Follow the steps below to install Argo CD using Helm:
...
+
+    ![Add Ons Overview](../assets/operator-guide/argocd-addons.png "Add Ons Overview")
 
-      controller:
-        enableStatefulSet: true
+     Once these components are installed, proceed with the Argo CD configuration.
 
+6. Update Argo CD helm chart:
+
+    To enable features such as ingress, login via oidc provider, provisioning secret using ESO need to update ArgoCD with [values](https://github.com/epam/edp-cluster-add-ons/blob/main/argo-cd/values.yaml) below:
```
Comment (daniil-nedostup) [inline docs/operator-guide/install-argocd.md:133]: Argo CD*

File: docs/operator-guide/deploy-aws-eks.md [hunk 2]
```diff
@@ -464,7 +464,11 @@ Please find the detailed description of the variables in the [argo-cd/variables.
 
     :::
 
+## Next Step
+
+  - Install [Argo CD](install-argocd) and integrate it with Edp Cluster Addons
```
Comment (daniil-nedostup) [inline docs/operator-guide/deploy-aws-eks.md:469]: lets replace "Edp Cluster Addons", for example:
-  Install [Argo CD](install-argocd) and integrate it with [edp-cluster-add-ons](https://github.com/epam/edp-cluster-add-ons/tree/main/argo-cd) components.
or 
- Install [Argo CD](install-argocd) and integrate it with the [edp-cluster-add-ons](https://github.com/epam/edp-cluster-add-ons) repository.

File: docs/operator-guide/deploy-aws-eks.md [hunk 3]
```diff
@@ -385,7 +385,7 @@ Please find the detailed description of the variables in the [argo-cd/variables.
 
     :::
 
-5. Once `AWSIRSA_Test_ArgoCDMaster` AWS IAM Role is successfully created, you can navigate to our [Cluster addons](../../add-ons-overview.md) to set up IRSA annotations for the Argo CD application:
+5. Once `AWSIRSA_Test_ArgoCDMaster` AWS IAM Role is successfully created, you can navigate to our [Cluster addons](add-ons-overview.md) to set up IRSA annotations for the Argo CD application:
```
Comment (alezander86) [inline docs/operator-guide/deploy-aws-eks.md:388]: you can navigate to our - replace - navigate to the

File: docs/operator-guide/install-argocd.md [hunk 4]
```diff
@@ -87,106 +87,107 @@ Follow the steps below to install Argo CD using Helm:
...
+    :::info
+      By default, to access the console with administrative privileges, use the following credentials:
+	    - URL: localhost:8080
+      -	Login: admin
+    :::
+
+    ```bash title="password"
+    kubectl get secret -n argocd argocd-initial-admin-secret --template={{.data.password}} | base64 -d
+    ```
+5. Integrate Argo CD with add-ons and install core add-ons:
```
Comment (alezander86) [inline docs/operator-guide/install-argocd.md:116]: 5. Integrate Argo CD with Add-Ons(add-ons-overview.md) and install core add-ons:

File: docs/operator-guide/install-argocd.md [hunk 5]
```diff
@@ -87,106 +87,107 @@ Follow the steps below to install Argo CD using Helm:
...
+	    - URL: localhost:8080
+      -	Login: admin
+    :::
+
+    ```bash title="password"
+    kubectl get secret -n argocd argocd-initial-admin-secret --template={{.data.password}} | base64 -d
+    ```
+5. Integrate Argo CD with [Add-Ons](add-ons-overview.md) and install core add-ons:
+
+    To ensure the system functions correctly, the following core components must be installed:
```
Comment (MykolaMarusenko) [inline docs/operator-guide/install-argocd.md:118]: system?

Comment (MykolaMarusenko) [reply docs/operator-guide/install-argocd.md:118]: let's provide more details here. So we recommend to use components like: eso (secure secrets provisioning) ,ingress (external endpoint enabled - ingres),kk (to enable sso approach),kk-operator (to configure kk instance).

Comment (MykolaMarusenko) [reply docs/operator-guide/install-argocd.md:118]: or maybe provide sections in values.yaml that required for these components enabled

File: docs/operator-guide/install-argocd.md [hunk 6]
```diff
@@ -87,106 +87,107 @@ Follow the steps below to install Argo CD using Helm:
...
+        oidc:
+          enabled: true
+        eso:
+          enabled: true
       ```
 
     </details>
 
-2. Run the installation:
+## Next Step
```
Comment (MykolaMarusenko) [inline docs/operator-guide/install-argocd.md:187]: maybe we need some result of this documentation? So now we have:
1) Centralized Argocd instance deployed by helm
2) add-ons repo enable and configured with argocd
3) Init componentnt (eso, kk,,) deployed

**Review by MykolaMarusenko — APPROVED** [review]

**Pull request merged**